src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user2 = new User();
+    private static User user3 = new User();
```

Comment:
I think It's better not to use static users in tests. Your unit tests must be independent of each other, and if you create static test data, the state of user2 and user3 can move from one test to another. It's better to make this parameters local, or initialize them in  BeforeEach test (so the state would be definitely new for every new test)
And it is bad practice to use numbers in variable names